CanalTP/artemis
# artemis: tests added on trip add chaining with some options

Tests on trip add with multiple actions:

1. Add a new trip, update with a delay and update to normal
2. Add a new trip, update with a delay, update with a stop_time deleted and update to normal  
3. Add a new trip, update with a delay, update with a new stop_time and delete de trip

For details see: https://jira.kisio.org/browse/NAVP-1068
